docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -20,12 +20,35 @@ For this purpose, *Fast DDS Statistics module* exposes an extended DDS |Statisti
 Enable statistics DataWriters
 -----------------------------
 
-Statistics DataWriters are enabled using the method |enable_statistics_datawriter|.
-This method requires as parameters:
+Statistics DataWriters can be enabled in different ways.
+First, the built-in creation option, by which, if a statistics topic is defined either on the `fastdds.statistics`
+property policy, or on the `FASTDDS_STATISTICS_ENVIRONMENT_VARIABLE` environment variable.
+If a statistics topic is defined on either of those ways, when creating a DomainParticipant with `auto-enable` option,
+the associate DataWriters will automatically be enabled, with the recommended statistic QoS.
```

Comment:
This information is already in the documentation in the previously mentioned section.